src/components/views/messages/MessageEvent.js

```diff
@@ -63,18 +63,24 @@ module.exports = React.createClass({
         };
 
         var content = this.props.mxEvent.getContent();
-        var msgtype = content.msgtype;
+        var msgType = content.msgtype;
```

Comment:
msgtype was deliberately being left with a little 't' here for consistency with the incorrectly capitalised `content.msgtype`.  The latter is part of the spec and is almost impossible to fix at this point, so rather than having to remember which msgtypes are lowercased and which are camelCased, we have been keeping them all lowercase.  Happy to change it if there's a good reason though?